cnxepub/tests/test_collation.py

```diff
@@ -280,4 +280,4 @@ def test_with_ruleset(self):
         # Check for the appended composite document
         self.assertEqual(len(collated_binder), 3)
         self.assertEqual(collated_binder[2].metadata['title'],
-                         'Composite One')
+                         'Document One')
```

Comment:
I read your commit message but I needed some extra explanation.  I understand that the composite pages will not have title overrides but what about normal pages?